foolbox/models/base.py

```diff
@@ -64,9 +64,11 @@ def _preprocess(self, inputs: ep.TensorType) -> ep.TensorType:
         min_, max_ = self._model.bounds
         return x * (max_ - min_) + min_
 
-    @property
-    def data_format(self) -> Any:
-        return getattr(self._model, "data_format", None)
+    def __getattr__(self, item: str) -> Any:
```

Comment:
I don't like that we overwrite `__getattr__` (and even less that we then call `__getattribute__`). Can we maybe stick with overwriting `data_format` and then use `raise AttributeError` or so if the `getattr` call on the model returns `None`?

Reply:
Yes.